src/itkScancoImageIO.cxx

```diff
@@ -1390,7 +1390,9 @@ ScancoImageIO::WriteISQHeader(std::ofstream * file)
     this->InitializeHeader();
   }
   this->SetHeaderFromMetaDataDictionary();
+  // now overwrite some values which we don't want taken from metadata
   this->SetVersion("CTDATA-HEADER_V1");
+  this->m_MuScaling = 0.0; // we don't want rescaling to occur on reading
```

Comment:
@mkuczyns is this a good way to accomplish no rescaling on future reading?

Reply:
This should work for ISQ images. But if we write out an ISQ using ITKIOScanco, re-import it to the Scanco system, convert it to AIM for processing, and then use ITKIOScanco to convert the AIM to another file format, the image will be rescaled. I did some testing with this yesterday and it looks like the Scanco system will accept an ISQ with muScaling = 0, but automatically converts muScaling to 1 for AIMs (just as a reference, the only muScaling values I've seen are 4096, 8192, and 1). I don't think Scanco allows AIMs with muScaling = 0. I think you can leave this as is, but we will need to change the if statement before scaling to HU (line 1025) to work with AIMs with muScaling = 1. (i.e., changing m_MuScaling >0 to >1 seems to work in my tests).